app/lib/frontend/email_sender.dart

```diff
@@ -41,7 +42,14 @@ class EmailSender {
     } else {
       _logger.info('Sending email: $debugHeader...');
       try {
-        await send(_toMessage(message), _server);
+        await retry(
+          () => send(
+            _toMessage(message),
+            _server,
+            timeout: Duration(seconds: 15),
+          ),
+          delayFactor: Duration(seconds: 2),
```

Comment:
Can we limit it retry `TimeoutException` and what other exception it threw last time? Using `retryIf: (e) => e is TimeoutException  || e is ...`

It's just nice to avoid retrying exception we don't know what are.

Reply:
IIRC, but I can't find it now, it was an `IOException`, and I think the current one is similar (at opening the connection). Should we filter on that?